Basic FE project template/ui-src/styles/components/index.scss

```diff
@@ -0,0 +1,6 @@
+body {
```

Comment:
why did you add code in an index.scss file? this should only contain file imports

Reply:
also it is not correct to add it on the body